### PR TITLE
[ticket/16279] Add permission for Emoji in topic title

### PR DESCRIPTION
**Pull request by 3D-I**
Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

https://tracker.phpbb.com/browse/PHPBB3-16279

### CODE REVIEW

File: phpBB/language/en/acp/permissions_phpbb.php [hunk 1]
```diff
@@ -79,6 +79,7 @@
 	'ACL_U_SAVEDRAFTS'	=> 'Can save drafts',
 	'ACL_U_CHGCENSORS'	=> 'Can disable word censors',
 	'ACL_U_SIG'			=> 'Can use signature',
+	'ACL_U_EMOJI'		=> 'Can use Emoji in topic title',
```
Comment (Crizz0) [inline phpBB/language/en/acp/permissions_phpbb.php]: Maybe better with plural? Emojis?

**Comment by Crizz0** [comment]
Maybe worth to think of a additional permission for the post text as well.

File: phpBB/language/en/acp/permissions_phpbb.php (hunk 1 above)
Comment (3D-I) [reply phpBB/language/en/acp/permissions_phpbb.php]: We already use `Emoji` elsewhere

immagine

**Comment by 3D-I** [comment]
I have used a wrong ticket ID in the commit messages. 😱 I will re submit a new PR.

**Comment by Eldenroot** [comment]
Would be really nice to cover all places in next PRs... not just for titles :)

**Comment by mrgoldy** [comment]
Emoji already fall under the smilies permission for post text.

**Pull request closed without merging**